src/core/connection.c

```diff
@@ -1770,6 +1770,11 @@ QuicConnStart(
 
     CXPLAT_TEL_ASSERT(Path->Binding == NULL);
 
+    QuicConnApplyNewSettings(
```

Comment:
We want to make sure we don't _also_ call this in QuicConnSetConfiguration for the client path now (but you do for server).

Reply:
Done.